### ISSUE TITLE
[Bug] Expenses for buying assets tracked incorrectly

### ISSUE THREAD

**Issue by DominiqueSite** [Eqonomize/Eqonomize#170]
Tl;dr: Eqonomize tracks expenses for investments in e.g. financial assets not as expenses, but as transfers. This means users cannot track these expenses under an expense account/category, leading to potentially dangerous confusion.

As an example, consider the following user story:
In March, Alice had an income of EUR 1000, and expenses of EUR 900. At the end of the month, Instead of transferring the leftover EUR 100 to a savings account or leaving it in the checking account for April, she wants to invest them in the stock market. For this, she
1. makes a *transfer* to the clearing account connected to her securities account, and then
2. proceeds to *buy* shares of an exchange-traded fund worth EUR 100.

In April, she has the same income/expense situation, but decides to instead
3. *transfer* EUR 100 to her savings account.

Suppose that after two years, the shares she bought have increased in value, so that she is able to sell them for EUR 120. As she does not want to continue her investment, she
4. *sells* the shares for EUR 120, and then
5. *transfers* EUR 120 from her clearing account to her savings account.

Note when writing this story in Eqonomize, steps 1, 3 and 5 are tracked accurately. Steps 2 and 4 though, even though they are similarly tracked as transfers between asset accounts, are actually also expenses/incomes that should be tracked as such via the corresponding accounts (i.e. expense/income categories). Tracking these transactions only as transfers does not allow Alice to differentiate between the balance sheets of March and April (even though her balance in March was neutral while her balance in April was positive by EUR 100). Leaning decisions on such false information supplied by the Eqonomize balance sheets can be dangerous, as she might make financially unwise decisions about spending her money in May, thinking she had previously saved up EUR 100, when in reality it was only EUR 100.
